src/core/transaction.js

```diff
@@ -1501,6 +1501,157 @@ Transaction.prototype.summary = function() {
   return txSummary;
 };
 
+/**
+ * Construct a 'SuspendedPaymentCreate' transaction
+ *
+ * Relevant setters:
+ *  - setSourceTag()
+ *  - setFlags()
+ *  - setDigest()
+ *  - setCancelAfter()
+ *  - setFinishAfter()
+ *
+ *  @param {String} source account
+ *  @param {String} destination account
+ *  @param {Amount} payment amount
+ */
+
+Transaction.prototype.suspendedPaymentCreate = function(options_) {
+  let options;
+
+  if (typeof options_ === 'object') {
+    options = _.merge({}, options_);
+
+    if (_.isUndefined(options.account)) {
+      options.account = options.src || options.from;
```

Comment:
If it is copy/pasted, then I think we should get rid of it because we don't need legacy support in new code.